plugins/zsync.koplugin/main.lua

```diff
@@ -20,7 +21,7 @@ local ZSync = InputContainer:new{
 
 function ZSync:init()
     self.ui.menu:registerToMainMenu(self)
-    self.outbox = self.path.."/outbox"
+    self.outbox = DataStorage:getFullDataDir().."/outbox"
```

Comment:
@NiLuJe @poire-z @pazos Or maybe zsync-outbox instead of just outbox?

Reply:
I'm still not sure what zsync is about :| so, I dunno what ends up in that directory. But indeed, better named as the thing that put stuff in it, than a generic "output" which says nothing.

Reply:
It puts shared files there for the duration of the sharing, so that they're transferred to devices that "subscribe". It can be useful to transfer a file if you have two relatively limited devices, or just because it doesn't require configuration. (That being said, I never use it.)

Reply:
In the way I wrote it now I guess it can't work on Android, which is its own can of worms.

https://github.com/koreader/koreader/blob/debbd447a6a14f4416d6c74990f2d94eb913d6c8/plugins/zsync.koplugin/main.lua#L146-L154

But I'm not sure if it currently actually works without crashing, see #3880.